org.osgi.security.sysatt110.test/src/org/osgi/security/sysatt110/test/SysAtt110Test.java

```diff
@@ -1,54 +1,47 @@
 package org.osgi.security.sysatt110.test;
 
-import org.osgi.framework.BundleContext;
+import static org.osgi.framework.FrameworkUtil.getBundle;
+
+import java.io.File;
+
+import org.junit.After;
 import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Test;
-import org.osgi.framework.BundleActivator;
-
+import org.osgi.framework.BundleContext;
 import org.osgi.security.util.api.Util;
 import org.osgi.util.tracker.ServiceTracker;
 
-import java.io.File;
-
 /**
  * 
  **/
 
-public class Activator implements BundleActivator
+public class SysAtt110Test
 {
-	private static BundleContext bundleContext;
-	private ServiceTracker serviceRef;
+	private BundleContext bundleContext = getBundle(SysAtt110Test.class).getBundleContext();
```

Comment:
It would be nice to have that that functionality integrated with  a JUnit Runner, otherwise it's just a "magic method". A JUnit runner would also avoid the need for a specific "Run as -> Bnd OSGi Test Launcher (JUnit)"